basket/news/tasks.py

```diff
@@ -761,213 +761,79 @@ def process_subhub_event_customer_created(data):
         sfdc.add(contact_data)
 
 
-@et_task
-def process_subhub_event_customer_updated(data):
-    """
-    Event name: customer.updated
-
-    Updates customer record in SFDC if payment processor/Stripe customer
-    changes their name or email address
-    """
-    statsd.incr('news.tasks.process_subhub_event.customer_updated')
-
-    first, last = split_name(data['name'])
-    contact_data = {}
-
-    # find user by their payment processor ID
-    user_data = get_user_data(payee_id=data['customer_id'])
-
-    # if user was found in sfdc, see if we should update their name(s)
-    if user_data:
-        # if current last name is '_', update it
-        if user_data['last_name'] == '_':
-            contact_data['last_name'] = last
-
-        # if current last name is blank/Null, update it
-        if not user_data['first_name']:
-            contact_data['first_name'] = first
-
-        if 'first_name' in contact_data or 'last_name' in contact_data:
-            sfdc.update(user_data, contact_data)
-    # if user wasn't found, add them
-    else:
-        contact_data['email'] = data['email']
-        contact_data['first_name'] = first
-        contact_data['last_name'] = last
-        contact_data['payee_id'] = data['customer_id']
-
-        # create the user in sfdc
-        sfdc.add(contact_data)
-        statsd.incr('news.tasks.process_subhub_event.customer_updated.created')
-
-
 @et_task
 def process_subhub_event_subscription_charge(data):
     """
-    Event names: invoice.finalized, payment_intent.succeeded
+    Event names: customer.subscription.created, customer.recurring_charge
 
     This method handles both new and recurring charges.
 
-    Each of the handled events contains different information on the charge
-    which will need to be combined.
-
-    The charge_id field (sent in both payloads) will tie the two events
-    together.
-
-    invoice.finalized *should* be sent first, which is missing the following
-    credit card related fields:
-
-    - brand
-    - last4
-    - exp_month
-    - exp_year
+    Each of the handled events contains the same payload data. The only variation below
+    is in regards to Initial_Purchase__c, which will be True for the
+    `customer.subscription.created` event, and False for the `customer.recurring_charge`
+    event.
     """
 
     statsd.incr('news.tasks.process_subhub_event.subscription_charge')
 
-    user_data = get_user_data(payee_id=data['customer_id'],
+    user_data = get_user_data(payee_id=data['customer'],
```

Comment:
I see at least 3 key names for the Customer ID in this code now: `customer`, `customer_id`, and `cust_id`. Is it possible to push for some naming consistency between events for common fields like this?